docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -10,6 +10,27 @@ The plugin provides functionality to test mobile application on real devices, An
 implementation(group: 'org.vividus', name: 'vividus-plugin-mobile-app', version: '{page-component-version}')
 ----
 
+
```

Comment:
I propose to define reliable strategies in locator selection.
https://appiumpro.com/editions/20-making-your-appium-tests-fast-and-reliable-part-2-finding-elements
https://appiumpro.com/editions/60-how-to-pick-the-right-locator-strategy
^^^
As a reference

Reply:
I'll add this once "all" the mobile locators are available in vividus

Reply:
But at least I consider it necessary to highlight XPath bad performance. And you could put a remark about locators Vividus doesn't support

Reply:
I agree with this, but if we highlight bad performance, we should provide an alternative, I mean it would be better if I add all this info with adding new locator which is better than this one

Reply:
It will be like "Please refer to Appium Pro link" to pick right locator strategy, and the end-user will be able to select between xpath and a new one locator

Reply:
> I propose to define reliable strategies in locator selection.

Once they are added, they will be described in this section.

> But at least I consider it necessary to highlight XPath bad performance. 

It makes sense, see the official doc as example: http://appium.io/docs/en/commands/element/find-elements/index.html#selector-strategies: `Search the app XML source using xpath (not recommended, has performance issues)`

> but if we highlight bad performance, we should provide an alternative

we shouldn't because the work is still in progress, we have announced anything yet